### ISSUE TITLE
warning: throw will always call terminate

### ISSUE THREAD

**Issue by ferdymercury** [lamyj/odil#69]
There is a compilation warning:

```
/opt/odil/src/odil/dcmtk/Exception.cpp: In member function ‘virtual const char* odil::dcmtk::Exception::what() const’:
/opt/odil/src/odil/dcmtk/Exception.cpp:58:50: warning: throw will always call terminate() [-Wterminate]
         throw std::runtime_error("Invalid source");
```

**Comment by ferdymercury** [comment]
I guess, it could be fixed by either removing the 'noexcept' keyword, or by doing instead
return "Invalid source";

**Comment by lamyj** [comment]
Good catch, thank you. Fixed in 492f07c.
